packages/presentation/src/PostMessageTelemetry.tsx

```diff
@@ -0,0 +1,36 @@
+import {useTelemetry} from '@sanity/telemetry/react'
+import {memo, useEffect, type FC} from 'react'
+import type {VisualEditingConnection} from './types'
+
+export interface PostMessageTelemetryProps {
+  comlink: VisualEditingConnection
+}
+
+const PostMessageDocumentVersions: FC<PostMessageTelemetryProps> = (props) => {
+  const {comlink} = props
+
+  const telemetry = useTelemetry()
+
+  useEffect(() => {
+    const isDev = process.env['NODE_ENV'] === 'development'
+    const debugTelemetry = process.env['SANITY_STUDIO_PRESENTATION_DEBUG_TELEMETRY'] === 'true'
```

Comment:
The way it's setup here would mean that the code we publish on npm has this:
```
process.env['SANITY_STUDIO_PRESENTATION_DEBUG_TELEMETRY'] === 'true'
```
It's a bit tricky, as we have people who embed their sanity studios on envs that do not have `process` available as a global.
And for certain systems, like next.js, it wants you to use this format:
```
process.env.SANITY_STUDIO_PRESENTATION_DEBUG_TELEMETRY === 'true'
```
Which our ts linting generally don't like.


To make this fully safe to use, regardless of bundlers and runtimes that will consume this library, you'd need to do something like:
```
const debugTelemetry = 
  (typeof process !== 'undefined' &&
   typeof process.env !== 'undefined') ?
process.env.SANITY_STUDIO_PRESENTATION_DEBUG_TELEMETRY === 'true' :
   false
```

Or better yet, we could use `debug` like they do in the studio 🤔

Reply:
We are now entirely leaning on Studio's telemetry implementation, and this looks to be handled here: https://github.com/sanity-io/sanity/blob/next/packages/sanity/src/core/studio/StudioTelemetryProvider.tsx#L30